### ISSUE TITLE
Methods without @Define() inside a processor throws error

### ISSUE THREAD

**Issue by kirtan403** [agent-ly/nestjs-agenda#2]
Hi,

While working with this module, I got an error. I attached the stack:

```
UnhandledPromiseRejectionWarning: TypeError: Cannot destructure property 'name' of 'this.metadataAcessor.getDefinitionOptions(...)' as it is undefined.
    at MapIterator.iteratee (D:\Development\project\node_modules\@agent-ly\nestjs-agenda\dist\agenda.explorer.js:43:25)
    at MapIterator.next (D:\Development\project\node_modules\iterare\lib\map.js:13:39)
    at FilterIterator.next (D:\Development\project\node_modules\iterare\lib\filter.js:12:34)
    at IteratorWithOperators.next (D:\Development\project\node_modules\iterare\lib\iterate.js:21:28)   
    at Function.from (<anonymous>)
    at IteratorWithOperators.toArray (D:\Development\project\node_modules\iterare\lib\iterate.js:180:22)
    at MetadataScanner.scanFromPrototype (D:\Development\project\node_modules\@nestjs\core\metadata-scanner.js:12:14)
    at D:\Development\project\node_modules\@agent-ly\nestjs-agenda\dist\agenda.explorer.js:40:34       
    at Array.forEach (<anonymous>)
    at AgendaExplorer.explore (D:\Development\project\node_modules\@agent-ly\nestjs-agenda\dist\agenda.explorer.js:34:14)
(node:1136) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:1136) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Based on the review of the code, I found it is something related to the method it is getting undefined. So, I tested and found the following details:

- I was having 2 methods in the processor file with @Processor()
- Only 1 method had @Define() and another method was called from the first method and was private.
- When I removed that It worked without any error.

So, based on this I found, I cannot use any method without a Define decorator in a processor class. Can we fix that? I can update to use a new service file, but sometimes it is better to write a private function handle the similar implementation in the same class.

**Comment by devlzcode** [comment]
Yea, that wasn't intended -- fixed in 1.2.0

**Comment by kirtan403** [comment]
Thanks
